ni/heroku.py

```diff
@@ -18,6 +18,10 @@ def contrib_auth_token():
         return os.environ['GH_AUTH_TOKEN']
 
     @staticmethod
+    def contrib_secret_token():
+        return os.environ['GH_SECRET_TOKEN']
```

Comment:
I guess catch the `KeyError`, use `log_exception()` and then bare `raise` it again.